Assets/Scripts/Game/UserInterface/FolderBrowser.cs

```diff
@@ -166,7 +166,22 @@ void Setup()
 
             // Setup initial folder conditions
             RefreshDrives();
-            RefreshFolders();
+            
+            if(DaggerfallUnity.Settings.PortableInstall)
+            { // start browsing from exe path for portable installs
+                currentPath = AppDomain.CurrentDomain.BaseDirectory;
+                if(SystemInfo.operatingSystemFamily == OperatingSystemFamily.Windows)
+                {
+                    var drive = Path.GetPathRoot(currentPath);
+                    driveList.SelectIndexSilent(drives.FindIndex( d => d == drive ));
+                }
+                UpdatePathText();
```

Comment:
I'm not sure this is behaving properly. 

![image](https://github.com/Interkarma/daggerfall-unity/assets/5789925/c6ed6efe-95db-43c9-8212-2e18fb4e63c9)

The ".." item is missing (probably because you made the "SelectIndex" silent and FolderList_OnUseSelectedItem could not add it). This probably needs to be added manually.